packages/flutter_driver/lib/src/driver.dart

```diff
@@ -384,6 +385,13 @@ class FlutterDriver {
     return GetTextResult.fromJson(await _sendCommand(new GetText(finder, timeout: timeout))).text;
   }
 
+  /// Sends a string and returns a string.
+  ///
+  /// The application can respond to this by providing a handler to [enableFlutterDriverExtension].
+  Future<String> requestData(String message, { Duration timeout }) async {
```

Comment:
I try to avoid words like Simple in APIs because if the API is simple, that should be obvious, and if it's not, the name will be a constant source of anger.

"customRequest()" doesn't really convey anything different than "requestData()" but is less verb-like, which is suboptimal for a method.

"requestWithData()" vs "requestData()" seems like a wash except the latter is shorter, so...

I dunno. I've left it as requestData() for now. I don't feel strongly about this though, feel free to change it.